api/api-elastic-profile-v1/src/main/java/com/thoughtworks/go/apiv1/elasticprofile/ElasticProfileControllerV1.java

```diff
@@ -75,6 +78,7 @@ public void setupRoutes() {
 
             get("", this::index);
             get(Routes.ElasticProfileAPI.ID, this::show);
+            get(Routes.ElasticProfileAPI.ID + "/usages", this::usages);
```

Comment:
can we add `mimeType`? This won't honor `Accept` header.

Reply:
Sure